java-api.md

```diff
@@ -199,6 +199,11 @@ Second generator would generate strings of length 8 from specified range of char
 "EF893232", "2E49D0AB", "BE129E15", "938FFC1C", "BB8A43ED", "829D1CA2", ...
 ```
 
+## Now methods
+
+These methods return current time, there are three variations: `nowDate()`, `nowLocalDate()` and `nowLocalDateTime()`.
```

Comment:
Date and time locality has no meaning without the context (visually presenting something to humans). I always prefer millis in UTC to any localized time. Localization is a problem of visualization (rendering) not data at this level. Just my 2 cents.